scripts/upgrade_ks_app.py

```diff
@@ -57,7 +57,7 @@ def main():
   registries = app['registries']
   libraries = app['libraries']
 
-  for name in registries.iterkeys():
+  for name in registries.keys():
```

Comment:
nit: can you do 
for name, _ in registries.items()

I think most linters prefer that.

Reply:
Done!